skfp/distances/all_bit.py

```diff
@@ -0,0 +1,365 @@
+from typing import Union
+
+import numpy as np
+from numba import njit, prange
+from scipy.sparse import csr_array
+from sklearn.utils._param_validation import validate_params
+
+from .utils import _check_nan
+
+
+@validate_params(
+    {
+        "vec_a": ["array-like", csr_array],
+        "vec_b": ["array-like", csr_array],
+    },
+    prefer_skip_nested_validation=True,
+)
+def all_bit_binary_similarity(
+    vec_a: Union[np.ndarray, csr_array], vec_b: Union[np.ndarray, csr_array]
+) -> float:
+    r"""
+    Calculate the all-bit binary similarity between two binary vectors.
+
+    Computes the all-bit similarity for binary data between two input arrays
+    or sparse matrices using the formula:
+
+    .. math::
+
+        sim(vec_a, vec_b) = \frac{\text{count}_a - \text{count}_\text{xor}}{\text{count}_a}
+
+
```

Comment:
Why so many empty lines? You don't need any after `math::`, and only 1 after the formula